src/tensor.c

```diff
@@ -132,11 +130,6 @@ RAI_Tensor *RAI_TensorCreateWithDLDataType(DLDataType dtype, long long *dims, in
         break;
     }
 
-    if (tensorAllocMode != TENSORALLOC_NONE && data == NULL) {
-        RedisModule_Free(ret);
-        return NULL;
-    }
-
```

Comment:
why you removed this?

Reply:
Because it checks that allocation succeeded, and we don't check this normally.